spec/models/tag_group_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe TagGroup, type: :model do
+  let(:project) { build :project, :with_tag}
+  let(:team) { build :team}
+
+  context "A project does have a tag_group" do
+    it { expect(project.tag_group.team_id).to eq(6) }
+    it { expect(project.tag_group.name).to eq('my-tag') }
+    it { expect(project.tag_group.description).to eq('awesome tag') }
+    it { expect(project.tag_group.bg_color).to eq('#FFFFFF') }
+  end
+
+  context "A team could have multiples tag_group" do
```

Comment:
You don't need to test Rails, if it's a `has_many` it'll always work as a `has_many`.